### ISSUE TITLE
blank page in dev.html

### ISSUE THREAD

**Issue by celiakessassi** [opensourceBIM/BIMsurfer#41]
Installed bimserverjar-1.5.150 with bimsurfer3-0.0.245.
The model is rendered in bimviews but not in bimsurfer 
![image](https://user-images.githubusercontent.com/10081364/61381689-bae5b100-a8ab-11e9-9895-63b6810ae78d.png)

**Comment by astashkevich39** [comment]
if I understood correctly, bimviews used an old version of BimSurfer 
![image](https://user-images.githubusercontent.com/29597786/61452635-8e3da200-a95c-11e9-86fc-5253e0c62700.png)

**Comment by rubendel** [comment]
Yes BIMvie.ws use**s** BIMsurfer V1. As for the original author, we really need more information, is it too much to ask for what's in the debug log? Also these settings seem kind of random, did you change them?

**Comment by astashkevich39** [comment]
In my case i have error in defaultrenderlayer.js. But in my custom code it do not used and all works fine
![image](https://user-images.githubusercontent.com/29597786/61455898-0f993280-a965-11e9-9f56-82dbdc2b6ac0.png)

**Comment by rubendel** [comment]
Object colors will be removed in the future and is not working anymore.

**Comment by astashkevich39** [comment]
That why setColor() do not work?

**Comment by rubendel** [comment]
Hijacking of issues is not appreciated.

**Comment by astashkevich39** [comment]
Sorry, got carried away.

**Comment by celiakessassi** [comment]
For me it not the same issue, No model is rendered in this page. Which part should I check ?

**Comment by rubendel** [comment]
The JavaScript console, hit F12 in most browsers. But your screenshot definitely shows that "useObjectColors" is enabled. This setting btw only specificies a specific way of loading object-colors, if you disable it colors will still work.

**Comment by rubendel** [comment]
Just to be clear, the dev.html is supposed to be used by BIMsurfer developers, who I would assume know where to find the JavaScript console. You should only change settings when you know what they mean. There a loads of combination that don't work together. The dev.html is mainly used for performance/functionality testing during development of BIMsurfer. Stick to the API if you just want to embed BIMsurfer.
